README.md

```diff
@@ -175,7 +176,12 @@ Run this command:
 
 You will also need to run this command whenever ``requirements.txt`` or ``test_requirements.txt`` change.
 
-#### 4) Create an Elasticsearch index
+#### 4) Create database tables
+To do this, run this command:
+
+    docker-compose run web ./manage.py migrate
+
+#### 5) Create an Elasticsearch index
```

Comment:
can you update the rest of the numbers in the section headers here? as-is there's two sections labelled `5` 😄